include/swift/AST/Attr.def

```diff
@@ -272,6 +272,8 @@ DECL_ATTR(_implements, Implements,
           | NotSerialized | UserInaccessible,
           /* Not serialized */ 67)
 
+DECL_ATTR(NSKeyedArchiveLegacy, NSKeyedArchiveLegacy, OnClass, 68)
```

Comment:
I definitely don't want to pollute the generated interface with mangled names, and I don't think it's worth us coming up with a way to print it without printing the mangled name.